openwisp_notifications/handlers.py

```diff
@@ -26,6 +27,7 @@
 logger = logging.getLogger(__name__)
 
 EXTRA_DATA = app_settings.get_config()['USE_JSONFIELD']
+EMAIL_BATCH_INTERVAL = app_settings.get_config()['EMAIL_BATCH_INTERVAL']
```

Comment:
The `app_settings.get_config()` returns the configuration for the Notification model. It is a django-notiifcations thing. 

Since you are defining the setting globally in openwisp_notifications/settings.py. You can access the setting like an attribute. 

```suggestion
EMAIL_BATCH_INTERVAL = app_settings.EMAIL_BATCH_INTERVAL
```